pkg/kritis/admission/admission.go

```
@@ -229,13 +235,56 @@ func createDeniedResponse(ar *v1beta1.AdmissionReview, message string) {
 
 func reviewImages(images []string, ns string, pod *v1.Pod, ar *v1beta1.AdmissionReview, config *Config) {
 	// NOTE: pod may be nil if we are reviewing images for a replica set.
-	glog.Infof("Reviewing images for %s in namespace %s: %s", pod, ns, images)
+	deny := false
+	errs := []string{}
+	glog.Infof("Reviewing Build Policy for images for %s in namespace %s: %s", pod, ns, images)
+	if err := reviewBuildPolicy(images, ns, config); err != nil {
+		deny = true
+		errs = append(errs, err.Error())
```

should we fail fast here?